nixos/tests/networking.nix

```diff
@@ -435,15 +435,15 @@ let
       nodes.client2 = node "192.168.1.2";
       testScript = { ... }:
         ''
-          startAll;
+          start_all()
 
           # Wait for networking to be configured
```

Comment:
all such comments would also be useful as subtests, because this way this explanation finds its way even into the output log and gives these actions context.

Reply:
Can you clarify on the semantics of what a subtest represents?
To me, this block looks like setup code that is necessary for the actual (sub-)test to succeed rather than test code itself. Wrapping that in subtest blocks seemed not intuitive to me.

Reply:
Subtests kind of "bundle" the outputs of groups of commands in a semantic group. Even in a test log it would be useful to see "oh, these lines are just for setup because the subtest says so".
You're right that setup is not really part of the test, but on the other hand it would be overkill to create even more library for such cases.

It's not a strong requirement that i would impose on anyone. But test logs *do* look much cleaner when they have most things in proper subtest groups.

Reply:
Thanks, updated the PR